tools/lang_batch_translate.py

```
@@ -0,0 +1,192 @@
+TARGET_LOCALES = [loc for loc in SUPPORTED_LOCALES if loc != SOURCE_LOCALE]
+
+CHINESE_RUN = re.compile(r"[\u4e00-\u9fff]+")
+TRANSLATOR_TARGETS = {"en_US": "en", "es_ES": "es", "ja_JP": "ja", "ko_KR": "ko", "zh_TW": "zh-TW"}
+
+
+def load_json(path: Path) -> dict:
+    try:
+        if path.exists():
+            return json.loads(path.read_text(encoding="utf-8"))
+    except Exception:
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)